ProcessMaker/bpm
# Feature/574 - Merge Laravel Horizon into UI

To test:
 - ensure you run composer install
 - ensure you run npm install
 - npm run dev

Url is at /manage/queues

Ensure laravel horizon UI renders as such
![horizoninsideprocessmaker](https://user-images.githubusercontent.com/3330334/44491709-fd105f00-a616-11e8-962c-4cf8e2a58eab.png)
